bin/cron/send_workshop_reminder_emails

```diff
@@ -1,26 +1,44 @@
 #!/usr/bin/env ruby
+
+# This script sends reminder via poste for professional development workshop
+# signups.
+
 require File.expand_path('../../../pegasus/src/env', __FILE__)
-require 'cdo/hip_chat'
-require 'cdo/solr'
 require 'cdo/poste'
 
-SOLR = Solr::Server.new(host: CDO.solr_server) if CDO.solr_server
-
-def main
-  workshops = SOLR.query(q: "*:*", fq: ["kind_s:ProfessionalDevelopmentWorkshop", "first_date_dt:[NOW/DAY+12DAY TO NOW/DAY+13DAY]"], sort: "id asc", rows: 1000)
-  workshops.each do |workshop|
-    signups = SOLR.query(q: "*:*", fq: ["kind_s:ProfessionalDevelopmentWorkshopSignup", "workshop_id_i:#{workshop['id']}", "-status_s:cancelled"], sort: "id asc", rows: 1000)
```

Comment:
Note that `workshop_id_i` is set [here](https://github.com/code-dot-org/code-dot-org/blob/staging/pegasus/forms/professional_development_workshop_signup.rb#L92), which in turn is set [here](https://github.com/code-dot-org/code-dot-org/blob/staging/bin/cron/process_forms#L85).